pkg/maintenance/alertmanagerMaintenance.go

```diff
@@ -21,6 +21,11 @@ var (
 	alertManagerRouteName          = "alertmanager-main"
 	alertManagerServiceAccountName = "prometheus-k8s"
 	alertManagerBasePath           = "/api/v2/"
+
+	// Generally upgrades should not fire critical alerts but there are some critical alerts that will fire.
+	// e.g. 'etcdMembersDown' happens as the masters drain/reboot and a master is offline but this is expected and will resolve.
+	// This is a regex of critical alerts that can be ignored while upgrading of controlplane occurs
+	controlPlaneIgnoredCriticalAlerts = "(etcdMembersDown)"
```

Comment:
Just calling out that this is silencing a etcd member down for potentially 90 mins but I don't know any way to refine it further other than what is called out in https://bugzilla.redhat.com/show_bug.cgi?id=1824986 which is probably to wait to alert after a certain amount of time down.